drivers/serial/uart_mcux_flexcomm.c

```diff
@@ -1067,6 +1068,33 @@ static int mcux_flexcomm_init(const struct device *dev)
 	return 0;
 }
 
+#ifdef CONFIG_PM_DEVICE
+static int mcux_flexcomm_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	const struct mcux_flexcomm_config *config = dev->config;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		if (!(config->base->CFG & USART_CFG_ENABLE_MASK)) {
+			/* Do not execute the Resume code if the UART is disabled. */
+			break;
+		}
+
+		break;
```

Comment:
This IF will be probably removed by the compiler. Did you forget to add resume code?

Reply:
I added this check in anticipation that resume code will be added in future.

Reply:
Does Zephyr allow comments that describe this kind of action if so I think we should have a comment indicating that related if checks should be added in this section of the code?